samples/tiny_http/src/main.rs

```diff
@@ -0,0 +1,58 @@
+extern crate tiny_http;
+extern crate multipart;
+
+use multipart::server::{Multipart, Entries, SaveResult};
+use tiny_http::{Response, StatusCode, Request};
+fn main() {
+    // Starting a server on `localhost:80`
+    let server = tiny_http::Server::http("localhost:80").unwrap();
+    loop {
+        // This blocks until the next request is received
+        let mut request = server.recv().unwrap();
+        process_multipart(&mut request).unwrap();
+
+        // Answering with an HTTP OK and a string
+        let response_string = "Multipart data is received!".as_bytes();
+        let response = Response::new(StatusCode(200),
+                                     vec![],
+                                     response_string,
+                                     Some(response_string.len()),
+                                     None);
+        request.respond(response).unwrap();
+    }
+}
+
+fn process_multipart(request: &mut Request) -> Result<(), Error> {
+    // Getting a multipart reader wrapper
+    let mut multipart = Multipart::from_request(request).unwrap();
```

Comment:
This will panic if the request was not a multipart request. I would prefer if the server just returned a boilerplate response and continued.